components/builder-api/src/config.rs

```diff
@@ -406,7 +391,7 @@ mod tests {
         assert_eq!(config.api.log_path,
                    PathBuf::from("/hab/svc/hab-depot/var/log"));
         assert_eq!(config.api.key_path,
-                   PathBuf::from("/hab/svc/hab-depot/files"));
+                   KeyCache::new("/hab/svc/hab-depot/files"));
```

Comment:
Perhaps not for this PR, but I think it would be nice to make a note here or decide on some future work to give `KeyCache` its own dedicated directory. This current directory contains non `KeyCache` type keys, for instance an RSA private key for the builder github app also exists here.

Reply:
I had that same thought... in practice, it won't really matter, as all Builder needs the `KeyCache` for is grabbing Builder encryption keys, but it would be nice to have that conceptual distinction.

Also, isn't that the directory where `hab file upload` files go?

Reply:
> Also, isn't that the directory where hab file upload files go?

That's exactly how the `.pem` file gets there. Also, I meant to add this comment to line 127 and not the test. Anyways, I think in the future we could change this to `/hab/svc/builder-api/var/keycahe` or something.